bridge/engine/ActiveMediaList.h

```diff
@@ -63,13 +63,16 @@ class ActiveMediaList
     }
 
     void onNewPtt(const size_t endpointIdHash, bool isPtt);
+    void mapSsrc2Usid(uint32_t ssrc, uint32_t usid);
 
     void process(const uint64_t timestampMs, bool& outDominantSpeakerChanged, bool& outUserMediaMapChanged);
 
     inline size_t getDominantSpeaker() const { return _dominantSpeakerId; }
 
     const std::unordered_set<size_t> getActiveTalkers() const;
 
+    utils::Optional<uint32_t> getUsid(const uint32_t ssrc);
```

Comment:
I do not understand the abbreviation. Can we spell it out?

Reply:
I renamed it ot getUserId() because I am hesitatant to put 'c9' into the name. But that would be an alternative: getC9userId().